tests/telemetry/events/eventd_events.py

```diff
@@ -8,7 +8,7 @@
 tag = "sonic-events-eventd"
 
 
-def test_event(duthost, gnxi_path, ptfhost, data_dir, validate_yang):
+def test_event(duthost, gnxi_path, ptfhost, ptfadapter, data_dir, validate_yang):
```

Comment:
Is `ptfadapter` referenced in this function?

Reply:
No, but test_event is a function defined in test_events.py as a function that will be called in all modules which takes in ptfadapter.